templates/taxi/manufacturer_list.html

```diff
@@ -23,7 +25,13 @@ <h1>Manufacturer List
               <td>
                   {{ manufacturer.country }}
               </td>
-          </tr>
+              <td>
+                <a href="{% url 'taxi:manufacturer-update' pk=manufacturer.id %}">Update</a>
+              </td>
+              <td>
+                <a href="{% url 'taxi:manufacturer-delete' pk=manufacturer.id %}">Delete</a>
```

Comment:
Consider adding a 'Cancel' button next to the 'Delete' button to allow users to navigate back to the previous page, as per the checklist requirement.